### ISSUE TITLE
PlanningScene Requesting initial scene failed

### ISSUE THREAD

**Issue by warmtan** [frankaemika/franka_ros2#12]
How to solve the problem? I tried to modify urdf, but could you solve the problem due to joint limitations

**Comment by christian-rauch** [comment]
There is an issue with concurrency I think. You have to wait until `You can start planning now!` appears in the CLI.

From the [documentation](https://support.franka.de/docs/franka_ros2.html#moveit):
> Wait until you can see the green `You can start planning now!` message from MoveIt inside the terminal. Then turn off the `PlanningScene` and turn it on again. After that turn on the `MotionPlanning`.

**Comment by BarisYazici** [comment]
Does this issue still persist?

**Comment by christian-rauch** [comment]
> Does this issue still persist?

Yes. `ros2 launch franka_moveit_config moveit.launch.py robot_ip:=dont-care use_fake_hardware:=true` still shows the `PlanningScene` warning `Requesting initial scene failed`.

Does it work for you without issues?

**Comment by BarisYazici** [comment]
No, I don't receive this error. Could it be related to the moveit version you are using. Could you try to use the .devcontainer we attached in the repo? Tell me if the warning persists.

**Comment by usmanbiu** [comment]
check this issue https://github.com/moveit/moveit2/issues/1907

**Comment by AndreasKuhner** [comment]
If there is still an open topic, please re-open the issue.
